.travis.yml

```diff
@@ -46,6 +46,10 @@ jobs:
     - name: Test Suite viewContent&metadata&extensions
       script: npm run build.e2e && SUITE="--suite viewer,infoDrawer,extensions" npm run e2e:docker
 
+    - stage: Trigger DW
+      if: (branch = master OR branch = development OR (tag =~ .*beta.*)) AND type = push
+      script: ./scripts/trigger-travis.sh --pro --branch $TRAVIS_BRANCH Alfresco digital-workspace $TRAVIS_ACCESS_TOKEN
```

Comment:
same comment as above regarding naming

Reply:
not sure I got but if is related to the one above what @popovicsandras is saying is correct

Reply:
Angular has the same but I guess they have a different pipeline...